src/apps/ubeswap/scripts/updateFarms.ts

```diff
@@ -0,0 +1,57 @@
+// Update the farms.json file with the latest farm info
+// This is a temporary solution as querying farm info from the blockchain is too slow
+// Medium term solution is to index farms using TheGraph
+// Usage: yarn ts-node ./src/apps/ubeswap/scripts/updateFarms.ts
+import fs from 'fs'
+import path from 'path'
+import { FarmInfoEventAbi } from '../abis/farm-registry'
+import { createPublicClient, http } from 'viem'
+import { celo } from 'viem/chains'
+
+const FARM_REGISTRY = '0xa2bf67e12EeEDA23C7cA1e5a34ae2441a17789Ec'
+const FARM_CREATION_BLOCK = 9840049n
+
+// How many blocks to query at once
+// Decrease this if the RPC node times out
+const BLOCKS_PER_QUERY = 500_000n
```

Comment:
fwiw, I think this is considered a pretty big chunk.  quicknode, for example, only allows 10_000 block sized chunks.